### ISSUE TITLE
The `--features` flag cannot be used multiple times

### ISSUE THREAD

**Issue by madsmtm** [rust-fuzz/cargo-fuzz#342]
Running the following fails, while it would succeed were I using a normal `cargo run`.

```sh
cargo +nightly fuzz run my_fuzz_target --features=foo --features=bar
```

I know I can just combine this into `--features=foo,bar`, but for composability it would be nice if `cargo-fuzz` didn't require me to do so.

**Comment by fitzgen** [comment]
Agreed that we should match `cargo`'s behavior here. Would be receptive to a PR that implemented this.
